packages/flutter_riverpod/lib/src/builders.dart

```diff
@@ -232,7 +232,40 @@ class ChangeNotifierProviderBuilder {
   ///   print(myParameter.userId);
   ///   print(myParameter.locale);
   ///   // Do something with userId/locale
-  /// })
+  /// });
+  ///
+  /// @override
+  /// Widget build(BuildContext context, ScopedReader watch) {
+  ///   int userId; // Read the user ID from somewhere
+  ///   final locale = Localizations.localeOf(context);
+  ///
+  ///   final something = watch(
+  ///     exampleProvider(MyParameter(userId: userId, locale: locale)),
+  ///   );
+  ///
+  ///   ...
+  /// }
+  /// ```
+  ///
+  /// ```dart
```

Comment:
We need to explain what this code block is about

Reply:
Take `userId` and `locale` externally and give them to the provider.

should be like this ?